mk/config.mk

```diff
@@ -513,11 +513,10 @@ endif
 # has a stable release.
 # This feature requires the support of Device Tree.
 CFG_TRANSFER_LIST ?= n
-ifeq ($(CFG_TRANSFER_LIST),y)
-$(call force,CFG_DT,y)
-$(call force,CFG_EXTERNAL_DT,y)
-$(call force,CFG_MAP_EXT_DT_SECURE,y)
-endif
+$(eval $(call cfg-enable-all-depends,CFG_TRANSFER_LIST, \
+	 CFG_DT CFG_EXTERNAL_DT CFG_MAP_EXT_DT_SECURE))
+
+CFG_TRANSFER_LIST_TEST ?= $(CFG_TRANSFER_LIST)
```

Comment:
A short description would be nice. Maybe mention that `CFG_TRANSFER_LIST_TEST` is not effective when `CFG_ENABLE_EMBEDDED_TESTS` is disabled.